es-space-mgmt/main/default/flows/createReservation.flow-meta.xml

```diff
@@ -288,13 +270,13 @@
         <showHeader>false</showHeader>
     </screens>
     <start>
-        <locationX>50</locationX>
-        <locationY>50</locationY>
+        <locationX>188</locationX>
+        <locationY>0</locationY>
         <connector>
             <targetReference>CustomerDetails</targetReference>
         </connector>
     </start>
-    <status>Active</status>
+    <status>Draft</status>
```

Comment:
You probably want to save the flow as `Active`.

Reply:
done!